app/queries/comments/community_wellness_query.rb

```diff
@@ -33,33 +33,34 @@ def self.sql_query
                 SELECT user_id,
                        COUNT(user_id) AS number_of_comments_with_positive_reaction,
                        /* Get the number of weeks, since today for posts */
-                       (trunc((extract(epoch FROM (current_timestamp- created_at))) / 604800) + 1) AS weeks_ago
```

Comment:
Taking away the `+ 1` in the truncate. Came to realization having week 0 (not award worthy) is indeed helpful